test/test_um2netcdf.py

```diff
@@ -206,124 +212,130 @@ def fake_out_path():
     return "/tmp-does-not-exist/fake_input_fields_file.nc"
 
 
+def mock_fix_time_no_time_dim(cube, verbose):
+    """
+    Side effect for fix_time_coord() mocks in process_cube() tests.
+    Replicates fix_time_coord()'s behaviour on cube's with no
+    time dimension.
+    """
+    return cube, None
+
+
 # FIXME: the convoluted setup in test_process_...() is a code smell
 #        use the following tests to gradually refactor process()
 # TODO: evolve towards design where input & output file I/O is extracted from
 #       process() & the function takes *raw data only* (is highly testable)
-def test_process_no_heaviside_drop_cubes(ta_plev_cube, precipitation_flux_cube,
-                                         geo_potential_cube, mule_vars, std_args,
-                                         fake_in_path, fake_out_path):
-    """Attempt end-to-end process() test, dropping cubes requiring masking."""
+def test_process_cubes_no_heaviside_drop_cubes(ta_plev_cube, precipitation_flux_cube,
+                                               geo_potential_cube, mule_vars, std_args,
+                                               fake_in_path, fake_out_path):
```

Comment:
```suggestion
def test_process_cubes_no_heaviside_drop_cubes(ta_plev_cube, precipitation_flux_cube,
                                               geo_potential_cube, mule_vars, std_args):
```

Remove unused test fixtures

Reply:
Good catch! I've removed these in bcef148